custom_conf.py

```diff
@@ -113,42 +115,59 @@
 
 # Set up redirects (https://documatt.gitlab.io/sphinx-reredirects/usage.html)
 # For example: 'explanation/old-name.html': '../how-to/prettify.html',
-
+# NOTE: If this variable is not defined, set to None or the dictionary is empty
```

Comment:
```suggestion
# NOTE: If this variable is not defined, set to None, or the dictionary is empty,
```
None of the commas is really required, but after reading this comment wrong for the 5th time now, I think they make it much easier to parse it correctly on the first try. ;)

Reply:
Note: I can see how this can be hard to read correctly; resolved with b3ba67c1eaa259a42b81d6c0667ae688435e9cd4